sdk/storage/azure-storage-blob/azure/storage/blob/shared_access_signature.py

```diff
@@ -0,0 +1,257 @@
+from azure.storage.blob._shared import sign_string, url_quote
```

Comment:
Could we please make this file private? (e.g. `_shared_access_signature.py`

Reply:
absolutely! Renamed ^_^